content/docs/csidriver/installation/helm/unity.md

```diff
@@ -101,8 +101,8 @@ Procedure
     **Note**: 
       * ArrayId corresponds to the serial number of Unity XT array.
       * Unity XT Array username must have role as Storage Administrator to be able to perform CRUD operations.
-      * If the user is using complex K8s version like "v1.21.3-mirantis-1", use below kubeVersion check in helm/csi-unity/Chart.yaml file.
-            kubeVersion: ">= 1.21.0-0 < 1.26.0-0"
+      * If the user is using a complex K8s version like "v1.24.6-mirantis-1", use this kubeVersion check in helm/csi-unity/Chart.yaml file.
+            kubeVersion: ">= 1.24.0-0 < 1.26.0-0"
```

Comment:
max version should be <1.27.0. please check it once.

Reply:
Done